packages/wallets/src/features/cfd/screens/ChangePassword/MT5ChangePasswordScreens.tsx

```diff
@@ -15,6 +16,36 @@ const MT5ChangePasswordScreens: React.FC<MT5ChangePasswordScreensProps> = ({ pla
     const handleClick = (nextScreen: TChangePasswordScreenIndex) => setActiveScreen(nextScreen);
 
     const { hide } = useModal();
+    const { data } = useSettings();
+    const { mutate } = useVerifyEmail();
+    const { data: activeWallet } = useActiveWalletAccount();
+
+    let redirectTo: number;
+
+    switch (platform) {
+        case 'mt5':
+            redirectTo = activeWallet?.is_virtual ? 11 : 10;
+            break;
+        case 'dxtrade':
+        default:
+            redirectTo = activeWallet?.is_virtual ? 21 : 20;
```

Comment:
Are we planning to hardcode these random numbers in the code?

Reply:
I believe this is what we have in production currently. I don't think there's a way for us to get the actual number anywhere. It was raised before by @hamza-deriv